docker-compose-auditor.yml

```diff
@@ -76,12 +76,15 @@ services:
       - SCALAR_DL_LEDGER_AUDITOR_ENABLED=true
 
   scalar-auditor:
-    image: ghcr.io/scalar-labs/scalar-auditor:3.9.1
-    # image: 709825985650.dkr.ecr.us-east-1.amazonaws.com/scalar/scalar-auditor:3.9.1
+    image: ghcr.io/scalar-labs/scalardl-auditor-byol:3.9.1
     container_name: "scalardl-samples-scalar-auditor-1"
     volumes:
       - ./fixture/auditor.pem:/scalar/auditor.pem
       - ./fixture/auditor-key.pem:/scalar/auditor-key.pem
+      - ./fixture/auditor.properties.tmpl:/scalar/auditor/auditor.properties.tmpl
+      - ./fixture/trial-license-cert.pem:/scalar/license-cert.pem
```

Comment:
At this time, the certificate file for validating a license key is fixed. So, I put it in this repository directly.